test/functional/forall/CMakeLists.txt

```diff
@@ -6,5 +6,7 @@
 ###############################################################################
 
 add_subdirectory(indexset)
+add_subdirectory(indexset-view)
```

Comment:
Looks like these are added here and in #793 - is that correct?

Reply:
Yup. I put up three PRs yesterday, each including the previous commits.  Probably not a great idea. I was trying to see if I could get the travis intel build to not timeout.  My last PR culls out a bunch of old tests that are covered in the new organization.

Reply:
That's fine, one good way to handle that kind of PR dependency is to have them merge into each other (e.g. set the target branch of each PR to the preceeding one) and only have the first one target develop.

Reply:
I thought about that, but too late. I'll change the base branches.